greeting-service/src/main/java/io/openshift/booster/cache/greeting/NameService.java

```diff
@@ -0,0 +1,56 @@
+/*
+ *
+ *  Copyright 2016-2018 Red Hat, Inc, and individual contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package io.openshift.booster.cache.greeting;
+
+import org.wildfly.swarm.spi.runtime.annotations.ConfigurationValue;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+
+/**
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
+ * <br>
+ * Date: 3/12/18
+ */
+@ApplicationScoped
+public class NameService {
+
+    @Inject
+    @ConfigurationValue("nameService.url")
```

Comment:
We typically don't use camelCase for configurables.

Reply:
Would name-service be better?